run-tests.sh

```diff
@@ -21,7 +21,6 @@ trap cleanup EXIT
 
 
 python -m check_manifest --ignore ".*-requirements.txt"
-python -m sphinx.cmd.build -qnNW docs docs/_build/html
```

Comment:
You should need to test build the docs, but you can remove the ``W`` flag if it causes too much troubles.